scripts/us_eia/opendata/process/common.py

```diff
@@ -21,7 +22,12 @@
 from sys import path
 
 # For import util.alpha2_to_dcid
-path.insert(1, '../../../../')
+# Setup path for import from data/util
+# or set export PYTHONPATH="./:<repo>/data/util" in bash
+#path.insert(1, '/usr/local/google/home/chandaluri/data/')
```

Comment:
Please remove if this line is not required.

Reply:
removed